src/Microsoft.PowerShell.Commands.Utility/commands/utility/ConvertFromMarkdownCommand.cs

```diff
@@ -66,12 +66,8 @@ public class ConvertFromMarkdownCommand : PSCmdlet
         /// </summary>
         protected override void BeginProcessing()
         {
-            _mdOption = this.CommandInfo.Module.SessionState.PSVariable.GetValue("PSMarkdownOptionInfo", new PSMarkdownOptionInfo()) as PSMarkdownOptionInfo;
-
-            if (_mdOption == null)
-            {
-                throw new InvalidOperationException();
-            }
+            _mdOption = this.CommandInfo.Module?.SessionState.PSVariable.GetValue("PSMarkdownOptionInfo", new PSMarkdownOptionInfo()) as PSMarkdownOptionInfo;
```

Comment:
I think this is not the right fix.
1. `Get/Set-MarkdownOption` both use the same code to set and get the option variable.
2. Need to make sure `Get/Set-MarkdownOption` continue to work even the utility.dll is loaded as snapin.

Reply:
Updated to use a concurrentdictionary